package.json

```diff
@@ -21,7 +21,7 @@
     "@salesforce/prettier-config": "^0.0.2",
     "@salesforce/ts-sinon": "1.4.6",
     "@swc/cli": "^0.1.62",
-    "@swc/core": "^1.3.39",
+    "@swc/core": "1.3.39",
```

Comment:
We removed the caret just to take the exact version of this library.

Peter: When regenerating yarn.lock the dep @swc/core@^1.3.39 picks up v 1.3.40. That version is somehow breaking snapshot:compare and generate. Pin @swc/core@1.3.39 at exact version.